docs/resources/rediscloud_active_active_subscription_database.md

```diff
@@ -85,7 +85,8 @@ output "us-east-2-private-endpoints" {
 The following arguments are supported:
 * `subscription_id`: (Required) The ID of the Active-Active subscription to create the database in. **Modifying this attribute will force creation of a new resource.**
 * `name` - (Required) A meaningful name to identify the database. **Modifying this attribute will force creation of a new resource.**
-* `memory_limit_in_gb` - (Required) Maximum memory usage for this specific database, including replication and other overhead
+* `memory_limit_in_gb` - (Deprecated) Maximum memory usage for this specific database, including replication and other overhead **Deprecated in favor of `dataset_size_in_gb`**
```

Comment:
Not a big deal but "(Deprecated)" is not consistent with the changes in `docs/resources/rediscloud_subscription_database.md` although I do prefer it to `Required`... I mean technically it's `Optional` now. Maybe mention the `you must provide one` constraint too, although I'm not sure how best to do any of that

Reply:
```suggestion
* `memory_limit_in_gb` - (Optional -  **Required if `dataset_size_in_gb` is unset**) Maximum memory usage for this specific database, including replication and other overhead **Deprecated in favor of `dataset_size_in_gb`**
* `dataset_size_in_gb` - (Optional - **Required if `memory_limit_in_gb` is unset**) The maximum amount of data in the dataset for this specific database is in GB
```
Hmm, what do you think of this?

Reply:
and the same changes would go for `docs/resources/rediscloud_subscription_database.md`

Reply:
Yeah, looks good to me!